source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1296,6 +1297,19 @@ message TransformationAdjustBranchWeights {
 
 }
 
+message TransformationChangingMemoryScope {
+
+  // This transformation is responsible for strengthens the memory scope
+  // operand within an atomic instruction.
```

Comment:
Change to: Widens a memory scope operand within an instruction.

Reply:
Done.